LocalPayment/src/test/java/com/braintreepayments/api/localpayment/LocalPaymentApiUnitTest.kt

```diff
@@ -0,0 +1,309 @@
+package com.braintreepayments.api.localpayment
+
+import android.net.Uri
+import com.braintreepayments.api.core.IntegrationType
+import com.braintreepayments.api.core.PostalAddress
+import com.braintreepayments.api.sharedutils.HttpResponseCallback
+import com.braintreepayments.api.testutils.Fixtures
+import com.braintreepayments.api.testutils.MockBraintreeClientBuilder
+import org.json.JSONException
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertSame
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.robolectric.RobolectricTestRunner
+import org.skyscreamer.jsonassert.JSONAssert
+
+@RunWith(RobolectricTestRunner::class)
+class LocalPaymentApiUnitTest {
+    private var localPaymentInternalAuthRequestCallback: LocalPaymentInternalAuthRequestCallback? =
```

Comment:
Nit - Similar change here, we could define this var as `lateinit` and make it non-null.